src/Aquifer.Common/Services/ResourceContentSearchService.cs

```diff
@@ -608,6 +617,9 @@ SELECT NULL
                             FROM ResourceContents rc2
                             WHERE
                                 rc2.ResourceId = rc.ResourceId AND
+                                {(filter.ExcludeContentMediaTypes is { Count: > 0 }
```

Comment:
This wasn't necessary for the PR but it was missing so I added it while I was in there.